Vagrantfile

```diff
@@ -37,7 +37,7 @@ fail 'Non-unique hostnames' if @network.collect { |_, v| v[:hostname] }.uniq!
 fail 'Non-unique ports' if @network.collect { |_, v| v[:ports].keys }.flat_map { |v| v }.uniq!
 
 def default_omnibus(config)
-  config.omnibus.chef_version = :latest
+  config.omnibus.chef_version = '12.17.44'
```

Comment:
Ran into this [error](https://gist.github.com/nmattam/4660e460eb0f40eb8c43c84c79e239db) while trying to vagrant provision on the stable branch. This error is thrown because _start recipe which creates the init.d is not executed and hence Splunk cannot restart. I was trying to isolate the error and see why _start is being skipped and it seems like the _configure_authentication is raising an exception (although this is not visible in the o/p - even in debug logging) and then it goes to execute all the queued resources (which is trying to restart splunk). If I comment out the _configure_authentication recipe I could successfully provision the nodes.
I locked chef-client to 12.17.44 and everything works fine. Looks like this has to do something with the 12.18.31(latest) version.